crates/biome_graphql_analyze/src/lint/nursery/use_deprecated_date.rs

```diff
@@ -0,0 +1,139 @@
+use biome_analyze::{
+    Ast, Rule, RuleDiagnostic, RuleSource, context::RuleContext, declare_lint_rule,
+};
+use biome_console::markup;
+use biome_graphql_syntax::GraphqlDirective;
+use biome_rowan::AstNode;
+use biome_rule_options::use_deprecated_date::UseDeprecatedDateOptions;
+use jiff::Zoned;
+
+declare_lint_rule! {
+    /// Require deletion date on `@deprecated` directive. Suggest removing deprecated things after deprecated date.
```

Comment:
```suggestion
    /// Require deletion date on `@deprecated` directive.
    ///
    /// Suggest removing deprecated things after deprecated date.
```

I don't understand the send phrase, can you rephrase it?